barretenberg/cpp/src/barretenberg/vm/tests/AvmMini_bitwise.test.cpp

```diff
@@ -101,80 +115,69 @@ class AvmMiniBitwiseNegativeTestsU128 : public AvmMiniBitwiseTests {};
 TEST_F(AvmMiniBitwiseTestsU8, BitwiseNot)
 {
-    // trace_builder
-    uint8_t input = 1;
-    trace_builder.set(input, 0, AvmMemoryTag::U8);     // Memory Layout: [1,0,0,...]
-    trace_builder.bitwise_not(0, 1, AvmMemoryTag::U8); // [1,0,255,0,0,....]
+    trace_builder.set(1, 0, AvmMemoryTag::U8);    // Memory Layout: [1,0,0,...]
+    trace_builder.op_not(0, 1, AvmMemoryTag::U8); // [1,0,254,0,0,....]
     trace_builder.return_op(1, 1);
     auto trace = trace_builder.finalize();
 
-    uint8_t res = ~input;
-    auto alu_row = common_validate_bitwise_not(trace, FF(1), FF(res), FF(0), FF(1), AvmMemoryTag::U8);
+    auto alu_row = common_validate_op_not(trace, FF(1), FF(254), FF(0), FF(1), AvmMemoryTag::U8);
 
     EXPECT_EQ(alu_row.aluChip_alu_u8_tag, FF(1));
     validate_trace_proof(std::move(trace));
 }
 
 TEST_F(AvmMiniBitwiseTestsU16, BitwiseNot)
 {
-    // trace_builder
-    uint16_t input = 1;
-    trace_builder.set(input, 0, AvmMemoryTag::U16);     // Memory Layout: [1,0,0,...]
-    trace_builder.bitwise_not(0, 1, AvmMemoryTag::U16); // [1,0,255,0,0,....]
+    trace_builder.set(512, 0, AvmMemoryTag::U16);  // Memory Layout: [512,0,0,...]
+    trace_builder.op_not(0, 1, AvmMemoryTag::U16); // [512,65023,0,0,0,....]
     trace_builder.return_op(1, 1);
     auto trace = trace_builder.finalize();
 
-    uint16_t res = ~input;
-    auto alu_row = common_validate_bitwise_not(trace, FF(1), FF(res), FF(0), FF(1), AvmMemoryTag::U16);
+    auto alu_row = common_validate_op_not(trace, FF(512), FF(65'023), FF(0), FF(1), AvmMemoryTag::U16);
 
     EXPECT_EQ(alu_row.aluChip_alu_u16_tag, FF(1));
     validate_trace_proof(std::move(trace));
 }
 
 TEST_F(AvmMiniBitwiseTestsU32, BitwiseNot)
 {
-    // trace_builder
-    uint32_t input = 1;
-    trace_builder.set(input, 0, AvmMemoryTag::U32);     // Memory Layout: [1,0,0,...]
-    trace_builder.bitwise_not(0, 1, AvmMemoryTag::U32); // [1,0,255,0,0,....]
+    trace_builder.set(131'072, 0, AvmMemoryTag::U32); // Memory Layout: [131072,0,0,...]
+    trace_builder.op_not(0, 1, AvmMemoryTag::U32);    // [131072,4294836223,,0,0,....]
     trace_builder.return_op(1, 1);
     auto trace = trace_builder.finalize();
 
-    uint32_t res = ~input;
-    auto alu_row = common_validate_bitwise_not(trace, FF(1), FF(res), FF(0), FF(1), AvmMemoryTag::U32);
+    auto alu_row = common_validate_op_not(trace, FF(131'072), FF(4'294'836'223LLU), FF(0), FF(1), AvmMemoryTag::U32);
 
     EXPECT_EQ(alu_row.aluChip_alu_u32_tag, FF(1));
     validate_trace_proof(std::move(trace));
 }
 
 TEST_F(AvmMiniBitwiseTestsU64, BitwiseNot)
 {
-    // trace_builder
-    uint64_t input = 1;
-    trace_builder.set(input, 0, AvmMemoryTag::U64);     // Memory Layout: [1,0,0,...]
-    trace_builder.bitwise_not(0, 1, AvmMemoryTag::U64); // [1,0,255,0,0,....]
+    trace_builder.set(0x100000000LLU, 0, AvmMemoryTag::U64); // Memory Layout: [8589934592,0,0,...]
+    trace_builder.op_not(0, 1, AvmMemoryTag::U64);           // [8589934592,0,255,0,0,....]
```

Comment:
Adapt the layout correctly. 255 should be removed and the output added at index 1.